nsv13/code/modules/squads/squad_vendor.dm

```diff
@@ -32,13 +32,19 @@
 	resistance_flags = ACID_PROOF | FIRE_PROOF
 	req_one_access = list(ACCESS_HOP, ACCESS_HOS)
 	var/static/list/loans_info = list()
-	var/static/list/loadouts = list()
+	var/loadout_type = /datum/squad_loadout/nt
+	var/list/loadouts = list()
 	var/static/list/requires_weapons_clearance = list(/obj/item/ammo_box, /obj/item/gun)
 
+/obj/machinery/squad_vendor/solgov
+	name = "Squad Vendor"
+	icon_state = "squadvend"
```

Comment:
```suggestion
```
These are inherited from the main type

Reply:
I was thinking about doing a sprite, but I think I'm going to leave it as is for now, so I'll remove it